classes/class3/README.md

```diff
@@ -1 +1,364 @@
+```javascript
+var http = require("http");
+
+http.createServer(function(request, response) {
+  response.writeHead(200, {"Content-Type": "text/plain"});
+  response.write("Hello World");
+  response.end();
+}).listen(8888);
+```
+
+Now let's see what the same app would look like using Express. To start we'll set up a new application using `npm init`. This will ask you a series of questions that will populate your `package.json` file, you can use the defaults for all of them. This file manages many things. If you publish your module on npm (like any of the modules which you `npm install` are published) this file will include the meta information about your module. We won't get into that much in this class, but that's why it seems like there are a lot of unnecessary fields. The main purpose we will use the package.json for is dependency management.
+
+After you exit the init setup, run `npm install express --save`. If you check your package.json again, you'll see the following was added:
+```javascript
+"dependencies": {
+    "express": "^4.10.6"
+  },
+```
+The `--save` argument will add the module to your list of dependencies and is generally a good thing to use whenever adding a new module. If you just forked a cloned a project that has its dependencies listed you can simple run `npm install` to install the required dependencies (at the right versions) for that project.
+
+Now that we're set up, create a file named `app.js` and and paste in the following:
+```javascript
+var express = require('express');
+var app = express();
+
+app.get('/', function(req, res){
+  res.send('hello world');
+});
+
+app.listen(3000);
+```
+Inside the folder you created the `app.js` file, run the command `node app` to start the server. Go to `http://localhost:3000` in your web browser and you should see the following:
+```
+hello world
+```
+Awesome! You just ran your first application using Express!
+
+Express makes writing web servers in Node much easier. Here are some of the important differences between the examples you did in the book and Express:
+* Routing. Instead of trying to parse the URL the user is at ourselves, we can just tell Express to match an individual path with a function.
+* Sending a response. Express takes care of setting many obvious response headers for you. Express will also handle sending files like images, music, audio, or `.html` files from a folder easily.
+* Handling templates. We haven't gotten here yet, but we'll touch on this later.
+Let's go back to our `app.js`. We want more than just hello world. Let's make Express show the string `hello olin` when we go to `http://localhost:3000/olin`.
+
+Below the `app.get('/',...` statement, add the following:
+```javascript
+app.get('/olin', function(req, res){
+  res.send('hello olin');
+});
+```
+
+So what does `app.get` do? It tells express that every time we have a `GET` request from a client (the browser). We'll learn more about this in Routing.
```

Comment:
It tells express what? The sentence just ended. 
It tells Express every time we have a `GET` request to execute the function which follows.